test/test-parse.cpp

```diff
@@ -539,6 +539,80 @@ TEST(test_escape_first_character_in_keyword,
   EXPECT_EQ(escape_first_character_in_keyword(u8"b1n z"_sv), u8"\\u{62}1n z");
   EXPECT_EQ(escape_first_character_in_keyword(u8"ZYXW"_sv), u8"\\u{5a}YXW");
 }
+
+#if QLJS_HAVE_SETJMP
+TEST(test_no_overflow, parser_depth_limit_not_exceeded) {
```

Comment:
This test should be enabled even if `QLJS_HAVE_SETJMP` is not supported.

Reply:
Done